pkg/server/sandbox_status.go

```diff
@@ -107,21 +107,24 @@ func toCRISandboxStatus(meta sandboxstore.Metadata, status sandboxstore.Status,
 				},
 			},
 		},
-		Labels:      meta.Config.GetLabels(),
-		Annotations: meta.Config.GetAnnotations(),
+		Labels:         meta.Config.GetLabels(),
+		Annotations:    meta.Config.GetAnnotations(),
+		RuntimeHandler: meta.RuntimeHandler,
 	}
 }
 
 // SandboxInfo is extra information for sandbox.
 // TODO (mikebrow): discuss predefining constants structures for some or all of these field names in CRI
 type SandboxInfo struct {
+	// Note: a new field `RuntimeHandler` has been added into the CRI PodSandboxStatus struct, and
+	// should be used. This `RuntimeHandler` field here will be deprecated after a grace period.
```

Comment:
/s/used/set/

Reply:
/s/here//

Reply:
pls move the documentation for the field down to the field.. vs pointing up..

Reply:
pls link to the actual deprecation if you have a link.

Reply:
Done! Thanks!